src/components/specs/Container.spec.js

```diff
@@ -0,0 +1,20 @@
+import { create } from 'react-test-renderer';
+import Container from '../Container';
+
+jest.mock('react-router-dom', () => ({
+  Link: 'Link',
+  Route: 'Route',
+  Switch: 'Switch',
+  useLocation: () => ({ pathname: '/1' }),
+}));
+jest.mock('../../hooks/useLaunchList', () => () => ({
+  launches: [{ flight_number: 1, launch_date_local: '2020-08-30T19:18:00-04:00' }]
+}));
+jest.mock('../../pages', () => ({ Dashboard: 'Dashboard', Details: 'Details' }));
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/megwali/spacex/pullRequest?prid=7693722)
```suggestion
jest.mock("../../pages", () => ({ Dashboard: 'Dashboard', Details: 'Details' }));
```